cdap-ui/app/cdap/components/SplashScreen/index.js

```diff
@@ -96,87 +85,71 @@ import T from 'i18n-react';
   toggleCheckbox() {
     this.doNotShowCheck = !this.doNotShowCheck;
   }
-  firstOnChange(e) {
-    this.setState({first : e.target.value});
-  }
-  lastOnChange(e) {
-    this.setState({last : e.target.value});
-  }
-  emailOnChange(e) {
-    this.setState({email : e.target.value});
-  }
   render() {
 
     let cardTitle = this.state.videoOpen ? '' : T.translate('features.SplashScreen.title');
+    let cardTitleTwo = this.state.videoOpen ? '' : T.translate('features.SplashScreen.titleTwo');
+
+    let cardHeader = (
+      <div className="card-header-splash">
+        <h3>{cardTitle}</h3>
+        <h3>{cardTitleTwo}</h3>
+        <span
```

Comment:
This will cause problem. Now there are two close buttons and the splash screen won't close because there is onClick handler on only one of them.